lib/src/interface/local_process_manager.dart

```diff
@@ -13,6 +13,7 @@ import 'dart:io'
         SYSTEM_ENCODING;
 
 import 'common.dart';
+import 'process_exception.dart';
```

Comment:
... and then this just gets imported from `dart:io` above.

Reply:
You're right.